pkg/trace/testutil/stats.go

```diff
@@ -19,7 +20,8 @@ const (
 )
 
 // BucketWithSpans returns a stats bucket populated with spans stats
-func BucketWithSpans(spans []*pb.Span) pb.ClientStatsBucket {
+func BucketWithSpans(spans []*pb.Span, conf *config.AgentConfig) pb.ClientStatsBucket {
```

Comment:
What is `BucketWithSpans` used for? Do we have any tests that use it and pass a specific config?

If we do have such tests, let's more explicitly pass just the `CustomTags` config rather than the whole `*config.AgentConfig`.
(EDIT: I see which test uses it, but I don't think we actually need to change that test.)

Otherwise if we don't, let's remove this change and just pass `nil` to `srb.HandleSpan()`.